XML3-Homework/Giles_Jan_26_Exercise_3.xml

```diff
@@ -0,0 +1,145 @@
+
+    <date>Last Updated: 2021/12/14 07:11 </date>
+    <p type="description"><character>Arataki Itto's</character>
+        <constellation>Taurus Iracundus</constellation> Story Quest is an upcoming story Quest for
+        version 2.3 of Genshin Impact. Learn how to unlock, walkthrough chart, release date, and
+        more about <constellation>Taurus Iracundus</constellation>!</p>
+
+    <subTitle><character>Arataki Itto</character> Story Quest - How To Unlock</subTitle>
+
+    <header><character>Arataki Itto</character> Story Quest</header>
+
+    <subHeader>A New Story Quest For 2.3 Update</subHeader>
+
+    <front>Release Date</front>
+
+    <end>
+        <date alt="2021/12/14">December 14, 2021</date>
+    </end>
+
+    <front>Eligibility</front>
+
+    <end>Adventure Rank 40 or above</end>
+
+    <front>How To Unlock</front>
+
+    <end>Complete the <quest>Omnipresence Over Mortals</quest></end>
+    <end>Complete Chapter: <quest>Act I - "Dracaena Somnolenta Chapter</quest>" </end>
+    <p type="description">Join <character>Arataki Itto</character> on his own story quest and get to
+        know more of <character>Itto's</character> background as the descendant of the Crimson Oni.
+        His story quest will be available permanently after the 2.3 game update. </p>
+    <p>Requires 1 Story Key To Unlock</p>
+
+    <p>Upon the release of the Story Quest, it's likely that it will require 1 story key to unlock
+        it similar to previously released Story Quests.</p>
+
+    <p>Get Story Keys By Doing Daily Commissions</p>
+
+    <p>You can obtain Story Keys by doing daily commissions. You can keep a maximum amount of 3
+        keys.</p>
+
+    <header>Who Is <character>Arataki Itto</character>?</header>
+
+    <subHeader>Oni of <location>Hanamizaka</location></subHeader>
+
+    <p type="characterDescription">A new <region>Inazuma</region>
+        <visionType>Geo</visionType> character that wields a <weaponType>Claymore</weaponType>
+        weapon. He is the first and considered the greatest Head of the Arataki Gang and an intrepid
+        man with Oni blood running through his veins.</p>
+
+    <header><character>Arataki Itto</character> Story Quest - Walkthrough Chart and
+        Location</header>
+
+    <subHeader>Story Quest Start Location</subHeader>
+
+    <p type="direction">Arataki Itto's story quest will start off by going to the
+            <location>Adventurer's Guild</location> and talking to <character>Katheryne</character>
+        of <region>Inazuma</region>. </p>
+    <subheader><character>Arataki Itto</character> Story Quest</subheader>
+
+    <p>Taurus Iracundus Chapter</p>
+
+    <p step="1">Talk to <character>Katheryne</character> in <region>Inazuma</region></p>
+
+    <p step="2">Talk to <character>Furuya Noboru</character></p>
+
+    <p step="3">Talk to <character>Ogura Mio</character></p>
+
+    <p step="4">Talk to <character>Uesugi</character></p>
```

Comment:
@julianjmg Is there a better tag than `<p>` for some of this? Are these paragraphs, really? Maybe `<instruction>`? Or something easier to spell, like `<in>`?